src/types/Comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
+  postId: number,
+  id: number,
+  name: string,
+  email: string,
+  body: string
+}
```

Comment:
Add empty lines at the end. Fix all cases
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file